### PR TITLE
Finished match history section

### PR DESCRIPTION
**Pull request by milesba4**

### CODE REVIEW

File: capstone-project/ui/src/components/MatchModal/MatchModal.jsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+export default function MatchModal({ matches, Match, toggleMatchModal }) {
+  const { matchModal, setMatchModal } = useAuthContext();
+
+  // filtering matches array to include only object where username is equal to username in onClick event
+  let UserInfo = matches.filter((match) => match.username == Match);
+  UserInfo = UserInfo[0];
+  // console.log("matchModal:", matchModal)
+  // console.log("confirmUsername:", Match)
+  console.log("userInfo:", UserInfo);
```
Comment (LifeBringer) [inline capstone-project/ui/src/components/MatchModal/MatchModal.jsx]: Comment out or delete

File: capstone-project/ui/src/components/ProfilePage/ProfilePage.jsx [hunk 2]
```diff
@@ -50,6 +51,21 @@ export default function ProfilePage() {
...
+
+  //if user toggles match modal
+  function toggleMatchModal(e){
+    setMatch((e.target.innerHTML).split(" |")[0])
+    setMatchModal(!matchModal)
+    console.log("e", e.target.innerHTML)
+    console.log("match:", match)
+    console.log("match:")
+    
+    console.log("works")
```
Comment (LifeBringer) [inline capstone-project/ui/src/components/ProfilePage/ProfilePage.jsx]: Comment out or delete x 4

**Review by LifeBringer — CHANGES_REQUESTED** [review]
See comments

File: capstone-project/ui/src/components/MatchModal/MatchModal.jsx (hunk 1 above)
Comment (gabremo) [reply capstone-project/ui/src/components/MatchModal/MatchModal.jsx]: Addressed!

File: capstone-project/ui/src/components/ProfilePage/ProfilePage.jsx (hunk 2 above)
Comment (gabremo) [reply capstone-project/ui/src/components/ProfilePage/ProfilePage.jsx]: Addressed!

Comment (milesba4) [reply capstone-project/ui/src/components/ProfilePage/ProfilePage.jsx]: Thanks, removed log statements

**Review by gabremo — APPROVED** [review]
Fantastic, Miles!!!! 🤯

**Review by LifeBringer — APPROVED** [review]

**Pull request merged**